src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2016-following-fast-tracks.md

```diff
@@ -493,6 +493,16 @@ in this case an index need to be migrated to the new Elasticsearch 5.x format:
 - The sequence index named `nuxeo-uidgen` cannot be migrated because the `_source` field is disabled, Nuxeo will take care to re-create this index at startup,
   so once this index has been backed up you can delete it.
 - The audit index named `nuxeo-audit` need to be migrated. Follow the [reindex upgrade procedure](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/reindex-upgrade.html).
+- in case the current `nuxeo-audit` index contains a `dueDate` field (Workflow usage), the format has changed from long to Date. This means the mapping in the target index should have the `dueDate` field type set to Date before reindexing. This could be done by adding the following to the nuxeo mapping or your custom one if needed:
+```
+    "dueDate": {
+        "type": "date"
+    }
+```
+If not migrated the field will generate errors like the following:
+```
+2018-09-18 10:39:40,100 ERROR [AuditLogWriterPool-00,in:168636,inCheckpoint:168617,out:0,lastRead:1537259979915,lastTimer:1537259979752,wm:201491740013363201,loop:1285532,record] [org.nuxeo.elasticsearch.audit.ESAuditBackend] Unable to index audit entry 16 :MapperParsingException[failed to parse [extended.dueDate]]; nested: NumberFormatException[For input string: "2018-09-19T10:39:39.486+02:00"];
+```
```

Comment:
https://doc.nuxeo.com/nxdoc/purging-audit-logs-nxp_logs/#1-create-a-new-audit-index
This dueDate mapping should be added into the mapping used in the upgrade/purging procedure https://doc.nuxeo.com/nxdoc/purging-audit-logs-nxp_logs/#1-create-a-new-audit-index

Reply:
done in master
https://github.com/nuxeo/doc.nuxeo.com-content/pull/814

but there is no 9.10 version of the documentation though, is there a PR for it?